lib/health/policies/exercise.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Health.Policies.Exercise do
```

Comment:
I wonder:

1.) Do we wanna make a global policies.ex file with each class's Bodyguard stuff in there

lib/health/policy.ex

2.) Or is it better to use a one policy file per class approach as I've sorta done here.

lib/health/policies/exercise.ex

Thoughts?

Reply:
I was thinking that it could be nice and manageable to have 1 policy file per context (what phoenix calls out Exercise, Weight files and their sub folders).

I guess I am envisioning it as if each context is like it's own API, with its own rules, and it's interface is through the top file only.

I'd really like to discuss this further, it is a pattern that I have been thinking about a lot.